apis/validator/attestation_data.yaml

```diff
@@ -4,7 +4,12 @@ get:
     - Validator
   operationId: "produceAttestationData"
   summary: "Produce an attestation data"
-  description: "Requests that the beacon node produce an AttestationData."
+  description: |
+    Requests that the beacon node produce an AttestationData.
+
+    A 503 error must be returned if the block identified by the response
```

Comment:
emm but user doesn't send `beacon_block_root ` in the request? 

Seems like we should just write here that the beacon node should never produce attestation for optimistic block root.